prow/config/config.go

```diff
@@ -921,6 +922,27 @@ type Gerrit struct {
 	// job runs for a given CL.
 	DeckURL        string                `json:"deck_url,omitempty"`
 	OrgReposConfig *GerritOrgRepoConfigs `json:"org_repos_config,omitempty"`
+	// allowedPresubmitTriggerRe is used to match presubmit test related commands in comments
+	allowedPresubmitTriggerRe          *CopyableRegexp
+	AllowedPresubmitTriggerReRawString string `json:"allowed_presubmit_trigger_re_raw_string,omitempty"`
+}
+
+func (g *Gerrit) GetAllowedPresubmitTriggerRegex() *CopyableRegexp {
+	return g.allowedPresubmitTriggerRe
+}
```

Comment:
We will never call this function without wanting to call `MatchString` immediately after so lets make this a matcher function like we do elsewhere rather than a getter function.

Reply:
Removed.